packages/orf/orf.1.0.0/opam

```diff
@@ -0,0 +1,60 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/orf"
+bug-reports: "https://github.com/UnixJunkie/orf/issues"
+dev-repo: "git+https://github.com/UnixJunkie/orf.git"
+license: "LGPL w/ linking exception"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "batteries"
+  "cpm"
+  "dolog"
+  "dune" {>= "1.10"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
The dune-project file is requiring 2.8.
I think a lower bound is needed in cpm or minicli, at least the lower bound checker is failing to compile in some cases.